tests/test_class_sh_property.cpp

```diff
@@ -0,0 +1,90 @@
+#include "pybind11_tests.h"
+
+#include "pybind11/smart_holder.h"
+
+#include <memory>
+
+namespace test_class_sh_property {
+
+struct ClassicField {
+    int num = -88;
+};
+
+struct ClassicOuter {
```

Comment:
Is the `Classic` prefix imply that this lass in not wrapped in `PYBIND11_SMART_HOLDER_TYPE_CASTERS`. A comment above each of these would help with readability.